guides/v2.1/contributor-guide/backward-compatible-development/index.md

```diff
@@ -189,6 +189,21 @@ Do not remove or rename {% glossarytooltip c57aef7c-97b4-4b2b-a999-8001accef1fe
 Do not change argument types.
 Instead of changing argument name or type, introduce new event argument with new name or type and deprecate the old argument by adding `@deprecated` annotation before dispatching the event.
 
+
+{% collapsible Example Code %} {% highlight php startinline %} /**
+
+$transportObject = new DataObject($transport);
+
+    /**
+     * Event argument `transport` is @deprecated. Use `transportObject` instead.
```

Comment:
Not sure it makes sense to use backticks in the comments, they have no meaning in PhpStorm.

Reply:
The DevDocs team plans to use a documentation generator for PHP API Reference that supports Markdown in the nearest future.